pkg/test/runner/runner.go

```diff
@@ -45,6 +45,11 @@ func NewRunner(testCase TestCase, c string) (*Runner, error) {
 
 // Run runs the test.
 func (r *Runner) Run() error {
+	if r.testCase.Config.Skip {
```

Comment:
Not familiar with the runner implementation but if it has access to the `testing.T` object, then probably using `t.Skip()` will be better.

Reply:
runner does not have access to it but the test that uses runner does. I have added some logic to use t.Skip()